codegen/module.go

```diff
@@ -851,13 +851,130 @@ func readPackageInfo(
 	}, nil
 }
 
-// GenerateBuild will, given a module system configuration directory and a
-// target build directory, run the generators assigned to each type of module
-// and write the generated output to the module build directory if commitChange
-func (system *ModuleSystem) GenerateBuild(
+func (system *ModuleSystem) getSpec(instance *ModuleInstance) (interface{}, bool, error) {
+	classGenerators := system.classes[instance.ClassName]
+	generator := classGenerators.types[instance.ClassType]
+
+	if generator == nil {
```

Comment:
There are examples I found:

```
Skipping generation of "vouchers" "endpoint" class of type "proxy" as generator is not defined
```

Reply:
gotcha, running in edge class proxy should implement incremental generator interface so it can gain benefits of incremental gen, but no strong opinion here